src/TodoList.js

```diff
@@ -1,5 +1,49 @@
 class TodoList {
+  constructor() {
+    this.items = [];
+    this.idCount = 1;
+  }
 
+  create(description) {
+    const todo = {
+      id: this.idCount,
+      text: description,
+      status: "incomplete",
+    };
+    this.idCount += 1;
+    this.items.push(todo);
+    return todo;
+  }
+
+  setToCompleted(id) {
+    for (let i = 0; i < this.items.length; i++) {
+      if (this.items[i].id === id) {
+        this.items[i].status = "completed";
+        return this.items[i];
+      }
+    }
+  }
+
+  getCompleted() {
+    let completed = this.items.filter((item) => item.status === "completed");
+    return completed;
+  }
+
+  getIncompleted() {
+    let incompleted = this.items.filter((item) => item.status === "incomplete");
+    return incompleted;
+  }
```

Comment:
Maybe you could combine these two together since the you are filtering through the same thing?